### ISSUE TITLE
SVG doesn't resize if it doesn't contain "viewBox" attribute

### ISSUE THREAD

**Issue by igordmn** [JetBrains/compose-jb#474]
Compose 0.3.1

SVG:
```
import androidx.compose.desktop.Window
import androidx.compose.foundation.Image
import androidx.compose.foundation.layout.requiredSize
import androidx.compose.runtime.remember
import androidx.compose.ui.Modifier
import androidx.compose.ui.platform.LocalDensity
import androidx.compose.ui.res.loadSvgResource
import androidx.compose.ui.unit.dp

fun main() = Window {
    val density = LocalDensity.current
    val svg = remember(density) {
        loadSvgResource(
            """
                <svg width="100" height="100" xmlns="http://www.w3.org/2000/svg">
                    <ellipse fill="#FF0000" cx="50" cy="50" rx="50" ry="50"/>
                </svg>
            """.trimIndent().byteInputStream(),
            density
        )
    }

    Image(
        svg, 
        contentDescription = null,
        modifier = Modifier.requiredSize(400.dp)
    )
}
```

If we add `viewBox="0 0 100 100"` to SVG then it will resize.

Probably it is a bug in Skia. Skia should determine viewBox by width/height.

**Comment by igordmn** [comment]
According to the specification, `viewBox` isn't required: https://www.w3.org/TR/SVG/coords.html#ViewportSpace

**Comment by olonho** [comment]
So is it something that to be reported to Skia issue tracker?

**Comment by olonho** [comment]
So is it something that to be reported to Skia issue tracker?

**Comment by igordmn** [comment]
> So is it something that to be reported to Skia issue tracker?

Yes, we need to write a reproducer in C++

**Comment by igordmn** [comment]
See https://bugs.chromium.org/p/skia/issues/detail?id=11144

**Comment by tonsky** [comment]
Patched in https://github.com/JetBrains/skia-build/releases/tag/m91-1a01201b77 (commit https://github.com/JetBrains/skia-build/commit/bf2313d6303adb515e7bb05b97a9076bff58eade)

**Comment by igordmn** [comment]
Fixed in 0.5.0-build224

**Comment by igordmn** [comment]
Still reproduced in 0.5.0-build225

**Comment by igordmn** [comment]
Fixed in 0.5.0-build245

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-6362">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
